examples/truffle/contracts/AbiTypes.sol

```diff
@@ -84,4 +84,26 @@ contract AbiTypes {
     }
     return buf;
   }
+
+  event ValueUint(uint8, uint16, uint32, uint64, uint128, uint256 indexed value);
+  event ValueInt(int8, int16, int32, int64, int128, int256 indexed value);
+
+  event ValueBool(bool);
+
+  event ValueBytes(string id, bytes, bytes6, address whoami);
+  event ValueArray(uint64[], int32[3]);
+
+  event ValueIndexed(string indexed, uint64[] indexed);
+
+  event Values(bytes32 indexed block, address sender) anonymous;
```

Comment:
Structs are not supported, right? Maybe also test a tuple?

Reply:
:+1:

Reply:
```
examples/truffle/contracts/AbiTypes.sol:91:13: TypeError: This type is only supported in the new experimental ABI encoder. Use "pragma experimental ABIEncoderV2;" to enable the feature.
  event Foo(S s);
```